dev/com.ibm.ws.cdi.extension.bce_fat/.classpath

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would have named this project something like `io.openliberty.cdi.4.0.internal.bce_fat`

Reply:
Hmm, I was following the existing layout of FATs where they're broken up by the area they test. Even though this is a 4.0 specific test, it's a new area so it seemed sensible to create a bucket for that area, rather than create a 4.0 bucket.

Reply:
So far, all of the existing buckets that do not have a version number in the name are for CDI 1.2 and above. There are separate buckets for CDI 2.0 and CDI 3.0 specific tests. e.g. `io.openliberty.cdi.3.0.internal_fat`

Reply:
Yuck :(